proto/src/determined/api/v1/experiment.proto

```diff
@@ -550,6 +550,30 @@ message PutExperimentResponse {
   // Note: no LaunchWarnings.
 }
 
+// Request to continue an experiment.
+message ContinueExperimentRequest {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "id" ] }
+  };
+  // Experiment ID to continue.
+  int32 id = 1;
+  // Experiment config (YAML) to merge with the experiment's config.
+  string override_config = 2;
+}
+
+// Request to continue an experiment.
+message ContinueExperimentResponse {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "experiment", "config" ] }
+  };
+  // The created experiment.
+  determined.experiment.v1.Experiment experiment = 1;
+  // The created experiment config.
+  google.protobuf.Struct config = 2;
```

Comment:
isn't config already a field inside of `experiment`?

Reply:
Removed ```config``` field

I did it because ```CreateExperimentResponse``` has it, not sure why it does